### PR TITLE
Fix newsletter for up to three local partners

### LINKED ISSUE #2041: Add support for multiple local partners
### Is your feature request related to a problem? Please describe.
We really want to be able to have multiple local partners at Thalia. And the website current does not support this.

### Describe the solution you'd like
We would like the ability to add multiple local partners.

### Motivation
We think it's useful to have multiple local partners so we can show our students what is available in Nijmegen.

### Describe alternatives you've considered
Only having one local partner, which does not make sense in the current partner climate.

### Additional context
The local partner is also shown in the newsletter.

### PR DESCRIPTION
**Pull request by wdoeland**
Closes #2041

### Summary
Not all local partners were shown in the newsletter. Now they are all shown.

### How to test
Steps to test the changes you made:
1. Create a new newsletter and preview it 'on the site'.
2. Make 0 partners local partner
3. Refresh and see 0 local partners in the newsletter
4. Do steps 2-3 with 0, 1, 2 and 3 partners and see 0, 1, 2 and 3 local partners in the newsletter.

### CODE REVIEW

**Review by SebastiaanSafeguard — CHANGES_REQUESTED** [review]
-

**Review by se-bastiaan — CHANGES_REQUESTED** [review]
With this approach any change to the partner logos as uploaded by an administrator could break versions of the newsletter when opened at a later time. I would suggest trying to include them by using base64 images.

**Comment by wdoeland** [comment]
> With this approach any change to the partner logos as uploaded by an administrator could break versions of the newsletter when opened at a later time. I would suggest trying to include them by using base64 images.

A few things on this:
- I think base64 images don't work in a lot of email clients. 
- Partners (and their images) are not deleted but they are just not set to be active anymore.
- I also don't think it's a big issue if an image (with an alt text!) doesn't display properly after a while in a weekly newsletter, does anyone reread these? It's much more important to make sure the email works properly now.

**Comment by pingiun** [comment]
> I think base64 images don't work in a lot of email clients.

I don't think that's true

File: website/newsletters/templates/newsletters/email.html [hunk 1]
```diff
@@ -180,29 +180,16 @@ <h2 style="{{ item_h2_style }}">{{ item.title.upper }}</h2>
...
-                        {% trans "our local partner"|upper %}
-                    </p>
-                </td>
-                {% endif %}
                 {% if main_partner and main_partner.is_active %}
                 <td align="center" style="color: black; font-size: 12px;">
                     <p style="margin: 10px; font-family: 'Gill Sans', 'Trebuchet MS', sans-serif;">
                         <a href="{{ main_partner.link }}">
-                            <img src="{% baseurl %}{% static 'newsletters/nedap-logo.png' %}" height="95" style="height: 95px;" alt="{{ main_partner.name }}"/>
+                            <img src="{{ main_partner.logo.url }}" height="95" style="height: 95px;" alt="{{ main_partner.name }}"/>
```
Comment (pingiun) [inline website/newsletters/templates/newsletters/email.html:192]: This should include the baseurl. Relative urls don't work in emails

**Comment by pingiun** [comment]
> * Partners (and their images) are not deleted but they are just not set to be active anymore

When an image is changed for a partner the url can also change. I don't think we delete the old image. What do you think @se-bastiaan should we just merge this?

**Comment by se-bastiaan** [comment]
Other small note: the text version of the newsletter has not been changed yet.

File: website/newsletters/templates/newsletters/email.html [hunk 2]
```diff
@@ -180,29 +180,16 @@ <h2 style="{{ item_h2_style }}">{{ item.title.upper }}</h2>
...
-                        {% trans "our local partner"|upper %}
-                    </p>
-                </td>
-                {% endif %}
                 {% if main_partner and main_partner.is_active %}
                 <td align="center" style="color: black; font-size: 12px;">
                     <p style="margin: 10px; font-family: 'Gill Sans', 'Trebuchet MS', sans-serif;">
                         <a href="{{ main_partner.link }}">
-                            <img src="{% baseurl %}{% static 'newsletters/nedap-logo.png' %}" height="95" style="height: 95px;" alt="{{ main_partner.name }}"/>
+                            <img src="{% baseurl %}{{ main_partner.logo.url }}" height="95" style="height: 95px;" alt="{{ main_partner.name }}"/>
```
Comment (se-bastiaan) [inline website/newsletters/templates/newsletters/email.html:192]: Can you use the thumbnail instead of the full image

Comment (se-bastiaan) [reply website/newsletters/templates/newsletters/email.html:192]: `{% thumbnail main_partner.logo THUMBNAIL_SIZE_MEDIUM fit=False %}`

File: website/newsletters/templates/newsletters/email.html [hunk 3]
```diff
@@ -212,6 +200,36 @@ <h2 style="{{ item_h2_style }}">{{ item.title.upper }}</h2>
         </table>
     </div>
     {% endif %}
+    {% if local_partners.all %}
```
Comment (se-bastiaan) [inline website/newsletters/templates/newsletters/email.html:203]:
```suggestion
    {% if local_partners.exists %}
```

Dit is een hele zware query die niet nodig is :)

**Review by se-bastiaan — APPROVED** [review]

**Pull request merged**